testapp.c

```diff
@@ -574,8 +573,16 @@ static pid_t start_server(in_port_t *port_out, bool daemon, int timeout) {
     }
 
     /* Yeah just let us "busy-wait" for the file to be created ;-) */
-    while (access(filename, F_OK) == -1) {
-        usleep(10);
+    useconds_t wait_timeout = 1000000;
```

Comment:
max wait time of 1s, right? I've already seen under valgrind that the unix socket can take a while. Lets just ham it to 10 seconds? 5 seconds?